### DISCUSSION TITLE
Keepass/KeepassXC support?

### DISCUSSION THREAD

**Discussion by boldandbusted** [getsops/sops#1668] [Ideas]
Howdy. I'm curious if anyone has made some path towards integrating a KeePass database and/or KeePassXC into the SOPS workflows? I'm thinking about how I might export environment variables out from KeePass databases (or a running KeePassXC instance with a localhost proxy). Thanks!

**Comment by felixfontein** [comment]
How would you expect that KeePassXC would work together with SOPS?

Also you could simply use keepassxc-cli (part of KeePassXC) to extract values from a KeeyPass database.

**Reply by boldandbusted** [reply]
Thanks for that question - I was thinking kinda backwards about SOPS. SOPS isn't a method to retrieve, store and otherwise manage secrets, per se. It uses PKI to encrypt and decrypt files, or portions of structured text files. Sorry for the noise.

**Comment by boldandbusted** [comment]
Closing...
